docs/TESTS.md

```diff
@@ -1,8 +1,15 @@
 # Tests
 
-The Purescript exercise test suites may be run from the exercise directory
-using the pulp tool.
+The PureScript exercise test suites may be run from the exercise directory
+using the spago tool:
 
-    cd exercism/project/directory/purescript/hello-world
-    bower install
-    pulp test
+    cd <purescript-exercism>/exercises/<exercise-type>/<exercise name>
+    spago test
+
+where <purescript-exercism> is the root directory where you checked out this
+repository.
```

Comment:
Students don't checkout this repo but work with exercises via the Exercism CLI. As such, this path is not referring to the repository's directory but the directory on a student's machine when using the exercism CLI. 

```suggestion
    cd ~/<exercism-workspace>/purescript/<exercise name>
    spago test

where <exercism-workspace> is the CLI's workspace directory.
```

Reply:
I'll update.

Reply:
When students check out, will they have `~/<exercism-workspace>/purescript/<exercise name>` or `~/<exercism-workspace>/<exercise name>`? I.e. is `purescript` part of the folder structure?

Reply:
`purescript` is part of the folder structure